sdk/agrifood/azure-agrifood-farming/README.md

```diff
@@ -16,7 +16,7 @@ Use FarmBeats client library for Python to do the following.
 To use this package, you must have:
 - Azure subscription - [Create a free account][azure_subscription]
 - Azure FarmBeats resource - [Install FarmBeats][install_farmbeats]
-- Python 2.7, 3.6 or later - [Install Python][python]
```

Comment:
not sure if you want to include the 2.7 disclaimer on the readmes:

## _Disclaimer_

_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_

Reply:
I will let the library owner decide if they want to add it.